swagger.yml

```
@@ -224,17 +224,18 @@ paths:
       parameters:
         - name: limit
           in: query
-          description: limit of results
+          description: The number of results to be retrieved per request.
           required: false
           type: number
+          default: 100
         - name: startAt
           in: query
-          description: last order id, don't put it when first requested it
+          description: The last order ID in the query. Omit when making an initial request.
           required: false
           type: string
         - name: createdAt
           in: query
-          description: last order createdAt, don't put it when first requested it
+          description: The last order's createdAt value. Omit when making an initial request.
```

I think we can try to be consistent here with the phrasing and make it less technical (createdAt vs created at, or date of creation)
```suggestion
          description: Date of creation on the last order. Omit when making an initial request.
```